src/signals/incident-management/containers/IncidentSplitContainer/styled.js

```diff
@@ -0,0 +1,109 @@
+import styled from 'styled-components';
+
+import { Heading, Label, RadioGroup, themeColor, themeSpacing, Row } from '@datapunt/asc-ui';
+
+import Button from 'components/Button';
+import InfoText from 'components/InfoText';
+
+export const StyledDefinitionList = styled.dl`
+  margin: 0;
+  display: grid;
+  grid-row-gap: 0;
+  padding-bottom: ${themeSpacing(4)};
+
+  @media (min-width: ${({ theme }) => theme.layouts.medium.max}px) {
+    grid-template-columns: 3fr 3fr;
+  }
+
+  @media (min-width: ${({ theme }) => theme.layouts.large.min}px) {
+    grid-template-columns: 3fr 2fr;
+  }
+
+  dt,
+  dd {
+    @media (min-width: ${({ theme }) => theme.layouts.medium.max}px) {
+      padding: ${themeSpacing(2)} 0;
+    }
+  }
+
+  dt {
+    color: ${themeColor('tint', 'level5')};
+    margin: 0;
+    font-weight: 400;
+  }
+
+  dd {
+    padding-bottom: ${themeSpacing(2)};
+    font-weight: 600;
+    width: 100%;
+  }
+`;
+
+export const StyledSubmitButton = styled(Button)`
+  margin-right: ${themeSpacing(5)};
+`;
+
+export const StyledRadioGroup = styled(RadioGroup)`
+  display: inline-flex;
+`;
+
+export const StyledLabel = styled(Label)`
+  align-self: baseline;
+
+  * {
+    font-weight: normal
+  }
+`;
+
+export const StyledHeading = styled(Heading)`
+  margin-bottom: 0;
+`;
+
+export const StyledWrapper = styled.div`
+  padding-top: ${themeSpacing(6)};
+  padding-bottom: ${themeSpacing(6)};
+`;
+
+export const FormWrapper = styled(Row)`
+  display: grid;
+
+  @media (min-width: ${({ theme }) => theme.layouts.large.max}px) {
+    column-gap: ${({ theme }) => theme.layouts.large.gutter}px;
+    grid-template-columns: 7fr 5fr;
+  }
+`;
+
+export const StyledGrid = styled.div`
+  display: grid;
+  grid-row-gap: ${themeSpacing(8)};
+`;
+
+export const StyledSelect = styled.div`
+  max-width: calc(100% - ${themeSpacing(50)});
+
+  strong {
+    font-size: 16px;
+  }
+`;
+
+export const StyledGridRow = styled.div`
```

Comment:
It looks this is an empty div. Can we use a plain div when there is no style involved?

Reply:
The reason I used this naming is that I am looking for one way to do spacing on our app, with the named divs this makes it more readable IMHO and gives you more context, removed for now.